### DISCUSSION TITLE
Hide/unhide date gaps in datetime series in Time series plots

### DISCUSSION THREAD

**Discussion by pyeh-seg** [grafana/grafana#62534] [Dashboards, Panels & Visualizations]
In time series plots, regardless of our datetime series, the X axis always show all dates. This causes date gaps in our plots. Is there a way to hide/unhide date gaps? This is especially annoying if we have date series that does not contain non business days.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
